### PR TITLE
docs: How to import Prisma Client into your app

### PR DESCRIPTION
**Pull request by janpio**
New page that can be linked from `prisma generate` success message

### CODE REVIEW

**Comment by millsp** [comment]
Would be great to document all the different import variants:

Regular Prisma Client:
TS, CJS, ESM

Edge Prisma Client:
TS, CJS, ESM

Deno Prisma Client
TS

We might not want all the possible ways, but just in case [here](https://prisma-company.slack.com/archives/C5Z9TH6N9/p1686231499931869?thread_ts=1686156600.390869&cid=C5Z9TH6N9) are a few more ways.

**Comment by janpio** [comment]
Happy to merge and PR review suggestions. I just created this as a dummy to start the conversation.

**Comment by ruheni** [comment]
I think we could repurpose existing pages in our docs for this content. Here are a few ideas (non-exhaustive) for pages we could use in our docs:

- [Working with Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient) (currently acts more of a landing page) or
- [Instantiating Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/instantiate-prisma-client)

**Comment by millsp** [comment]
Reminder for the code related to this PR: We will need to create a link that points to `http://pris.ly/d/importing-client` that will be shown after the data proxy improvements PR is merged.

File: content/200-concepts/100-components/02-prisma-client/index.mdx [hunk 1]
```diff
@@ -92,6 +100,38 @@ const prisma = new PrismaClient()
 // use `prisma` in your application to read and write data in your DB
 ```
 
+</tab>
+
+</TabbedContent>
+
+For edge environments, you can import Prisma Client for edge environments as follows:
```
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/index.mdx]:
```suggestion
For edge environments, you can import Prisma Client as follows:
```

**Review by jharrell — APPROVED** [review]

**Comment by millsp** [comment]
I already added the shortlink here https://github.com/prisma/pris.ly/pull/124

**Comment by janpio** [comment]
The resulting page does not document how to use a custom `output` or how to import with Deno. (= missing documentation)
The headline "Importing Prisma Client" now also covers how to use it to send queries. (= worse than before documentation)

**Pull request merged**